sql/src/test/scala/org/datasyslab/geosparksql/adapterTestScala.scala

```diff
@@ -132,6 +133,12 @@ class adapterTestScala extends TestBaseScala {
       assert (df.columns(1) == "STATEFP")
     }
 
+    it("Read GeoJson that contains id"){
+      val rdd = GeoJsonReader.readToGeometryRDD(sparkSession.sparkContext, "sql/src/test/resources/idContainsGeoJson.json", false, false)
+      val df = Adapter.toDf(rdd, sparkSession)
+      df.show(false)
```

Comment:
Similarly, this test case doesn't actually test the correctness. Please use `assert` and `count` function to break Spark's lazy evaluation and check the correctness.